### DISCUSSION TITLE
Set Display for Performance ?

### DISCUSSION THREAD

**Discussion by Sniper199999** [ChrisTitusTech/winutil#181] [Q&A] [answered]
In "Tweak > Misc. Tweaks" what does "Set Display for Performance" do exactly and what does it change?

**Comment by Lakshaya-wq** [comment, marked as answer]
"Set Display To Performance" basically changes some windows transition/animation (Visual Effects) settings.

![image](https://user-images.githubusercontent.com/66407695/191900989-eac9e390-08e2-47b0-8d08-00cca7ed1fcc.png)

![image](https://user-images.githubusercontent.com/66407695/191901073-ff061c4c-dc2c-4cc8-9fe6-52e54939ff28.png)

**Reply by Sniper199999** [reply]
Got it, thank you for clarifying.

**Comment by BMP5000** [comment]
How do I revert the change?
The control panel settings are not working
